### PR TITLE
Add test component for multiple formats support

### PR DESCRIPTION
**Pull request by vpereira**
As asked by @joelhawksley on issue #1062, I'm opening a PR with a test component with multiple formats (html and text)

Feel free to guide me if something is missing.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ title: Changelog
 
     *Javi Martín*
 
+* Add test component for multiple formats support
+    * Victor Pereira*
```
Comment (joelhawksley) [inline docs/CHANGELOG.md:19]:
```suggestion
* Add test component for multiple formats support.

    * Victor Pereira*
```

File: test/view_component/multiple_formats.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ViewComponentMultipleFormatsTest < ViewComponent::TestCase
+  def test_render_html
+    render_inline(MyComponentMultipleFormat.new, format: :html)
```
Comment (joelhawksley) [inline test/view_component/multiple_formats.rb:7]: Wouldn't the format be set by the request? I'm guessing we'd need to add a `with_request_format` test helper.

**Review by joelhawksley — COMMENTED** [review]
Thanks for adding this test case! We should pair on making it pass 😄 

Email me: joelhawksley@github.com

File: test/view_component/multiple_formats.rb (hunk 2 above)
Comment (vpereira) [reply test/view_component/multiple_formats.rb:7]: Ok, I started with a skeleton code/pseudo code.

What I'm thinking to do is:

- check if there is a view for `#{component}.#{format}.#{extension}`. if it exists, set it as view.

**Comment by vpereira** [comment]
@joelhawksley any reason why `view_component` isn't using https://github.com/rails/rails/blob/main/actionview/lib/action_view/lookup_context.rb to find the templates?

**Comment by joelhawksley** [comment]
🤦🏻 I had completely forgotten about this PR and basically re-implemented it with @jaredmoody! For now, I'm going to close this as stale as we are tracking the need for this feature here: https://github.com/ViewComponent/view_component/issues/1670.

**Pull request closed without merging**